### PR TITLE
Upgrade to KeyCloak v1.17.1

### PR DESCRIPTION
**Pull request by stack72**
Also includes upgrade to:

* Pulumi v1.12.1
* Pulumi-terraform-bridge v1.8.2

### CODE REVIEW

File: examples/realm/ts/tsconfig.json [hunk 1]
```diff
@@ -13,6 +13,6 @@
         "forceConsistentCasingInFileNames": true
     },
     "files": [
-        "index.ts"
+      "ts/index.ts"
```
Comment (mikhailshilkov) [inline examples/realm/ts/tsconfig.json]: Why `ts` here? It's already in the `ts` folder.

File: go.mod [hunk 2]
```diff
@@ -5,13 +5,13 @@ go 1.13
...
+	github.com/mrparkers/terraform-provider-keycloak => github.com/pulumi/terraform-provider-keycloak v0.0.0-20200321130755-dfe55d7eb038
 )
 
 require (
 	github.com/hashicorp/terraform-plugin-sdk v1.7.0
 	github.com/mrparkers/terraform-provider-keycloak v0.0.0-00010101000000-000000000000
 	github.com/pkg/errors v0.9.1
-	github.com/pulumi/pulumi v1.9.1
-	github.com/pulumi/pulumi-terraform-bridge v1.6.4
+	github.com/pulumi/pulumi v1.12.1
```
Comment (mikhailshilkov) [inline go.mod]: `1.12.1` has that issue with python docs generation that fails the release. Maybe start using `1.13.0` or the same master-based that I used for other providers.

File: examples/realm/ts/tsconfig.json (hunk 1 above)
Comment (stack72) [reply examples/realm/ts/tsconfig.json]: fixed! Good spot - refactoring tools eh ;)

File: go.mod (hunk 2 above)
Comment (stack72) [reply go.mod]: nice tip!

**Pull request merged**